package ult.nodo.ciencia.excepcion;

public class AutorException extends GeneralException {

    public AutorException(Throwable cause) {
        super(cause);
    }
}
